find_max.rb

```diff
@@ -0,0 +1,19 @@
+
+# Find the maximum element between index 0 and max_index, exclusive
+
+def find_max(array, max_index)
+  max = nil
+  if max_index == 0
+    puts "Array is empty"
```

Comment:
If max_index is 0, doesn't that just mean that there won't be a max number? I don't think that the max_index being zero means that the array is empty.